client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1950,6 +1951,22 @@ public void testGetTemplateRequest() throws Exception {
         assertThat(request.getEntity(), nullValue());
     }
 
+    public void testAnalyzeRequest() throws Exception {
+        AnalyzeRequest indexAnalyzeRequest = new AnalyzeRequest()
+            .text("Here is some text")
+            .index("test_index")
+            .analyzer("test_analyzer");
+
+        Request request = RequestConverters.analyze(indexAnalyzeRequest);
+        assertThat(request.getEndpoint(), equalTo("/test_index/_analyze"));
+        assertThat(request.getEntity(), notNullValue());
```

Comment:
I think other folks are using ` assertToXContentBody(validateQueryRequest, request.getEntity());`. Might be worth leaving a comment about why you aren't using it so no one gets confused.